doc/how_to/make_a_sorting.rst

```diff
@@ -0,0 +1,89 @@
+Make your own Sorting
+=====================
+
+Why make a :code:`Sorting`?
+
+The :code:`Sorting` object is one of the core objects within the SpikeInterface library
+along with :code:`Recording` and :code:`SortingAnalyzer`. Although SpikeInterface has
+many tools for reading sorting formats you may have some data in a nonstandard format
+(e.g. old csv file). If this is the case you would need to make your own :code:`Sorting`.
+
+At a fundamental level the :code:`Sorting` is a series of spike times and a series of
+labels for each spike along with some associated metadata. Thus by providing this
+information you can easily make a :code:`Sorting` object to be used for various analyses
+(e.g. correlograms or for generating a :code:`SortingAnalyzer`)
+
+
+Making a :code:`Sorting`
+------------------------
+
+For most formats the :code:`Sorting` is automatically generated. For example one could do
+
+.. code-block:: python
+
+    from spikeinterface.extractors import read_kilosort, read_phy
+
+    # For kilosort/phy files we can use either reader
+    ks_sorting = read_kilosort('path/to/folder')
+    phy_sorting = read_phy('path/to/folder')
+
+This :code:`Sorting` contains important information about your spike trains including
+the spike times (i.e. when the neurons were actually firing) the unit labels (i.e.
+who the spikes belong to. Also called cluster ids by some sorters), the unit ids (the unique
+set of unit labels) and the sampling_frequency. To make your own :code:`Sorting` object you can
+use :code:`NumpySorting`. It is important to note if you are new to spike trains that they
+are typically stored in samples/frames rather than in seconds. So you should input the times
+in samples/frames. The sampling_frequency allows for easily switching between samples and seconds.
+
+There are 3 options (along with making a NumpySorting from another sorting which will not be covered here):
+
+With lists of spike trains and spike labels
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this case we need a list or array (or lists of lists for multisegment) of spike times,
```

Comment:
```suggestion
In this case we need a list or array (or lists of lists for multisegment) of spike frames,
```

I think that `times_list` is a bad argument name as it is prone to confusion.

Reply:
I agree. I would think we should say 

spike times (in frames) or spike times in frames instead. Saying spike times is common in the field to mean in frames or in seconds so I think it would be more standard in the field to say spike times and then specify the units. What do you think?

Reply:
I think that you are in a better position to make that call than me.  I trust you in this one.